index.js

```diff
@@ -2,8 +2,7 @@
 
 module.exports = YahooFantasy;
 
-var OAuth = require('oauth').OAuth,
-  https = require('https'),
+var request = require('request'),
```

Comment:
I prefer to `request` module, but I will try `https` module today and tell you some result.